perception/shape_estimation/launch/shape_estimation.launch.xml

```diff
@@ -1,18 +1,13 @@
 <launch>
   <arg name="input/objects" default="labeled_clusters"/>
   <arg name="output/objects" default="shape_estimated_objects"/>
-  <arg name="use_filter" default="true"/>
-  <arg name="use_corrector" default="true"/>
   <arg name="node_name" default="shape_estimation"/>
-  <arg name="use_vehicle_reference_yaw" default="false"/>
-  <arg name="use_vehicle_reference_shape_size" default="false"/>
-  <arg name="use_boost_bbox_optimizer" default="false"/>
```

Comment:
Since those args are deleted here, the arguments passed from the higher-level 'aunch (e.g. [lidar_based_detection.launch.xml](https://github.com/autowarefoundation/autoware.universe/blob/main/launch/tier4_perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml#L96-L97) are no longer being reflected."
Since 'args' is being cleared here, the arguments passed from the higher-level 'launch' are no longer being reflected."

Reply:
Got it.